src/scripts/_sandbox.js

```diff
@@ -4,8 +4,3 @@
 //
 // Do not commit changes to this file! In fact, you should locally set the `--assume-unchanged`
 // flag on it: `git update-index --assume-unchanged ./src/scripts/_sandbox.js`
-import 'module-alias/register'
-
-/**
- * DO NOT MODIFY ABOVE THIS BLOCK
- */
```

Comment:
Did you mean to remove the "DO NOT MODIFY ABOVE THIS BLOCK" comment?

Reply:
I did -- it doesn't matter what people do to that file since it (shouldn't be) tracked.

It used to be the case that we needed that import for anything to work correctly from sandbox.